mission_1.css

```
@@ -0,0 +1,262 @@
+*{
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+}
+header{
+  width: 1920px;
+  height: 70px;
+  padding: 10px 200px 9px 200px;
+  gap: 0px;
+  border: 0px 0px 1px 0px;
+  opacity: 0px;
+  background: #FFFFFF;
```

축약형으로 작성 가능한 값은 축약형으로 작성해주세요~! 
```suggestion
  background: #fff;
```